wrangler-service/src/main/java/co/cask/wrangler/service/connections/ConnectionTypeConfig.java

```diff
@@ -18,26 +18,32 @@
 
 import co.cask.cdap.api.Config;
 import co.cask.wrangler.dataset.connections.Connection;
+import com.google.gson.annotations.SerializedName;
 
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
+import javax.annotation.Nullable;
 
 /**
  * Default connections to create at startup and connection types that needs to be disabled
  */
 public class ConnectionTypeConfig extends Config {
   private final Set<ConnectionType> disabledTypes;
   private final List<Connection> connections;
+  @SerializedName("default")
```

Comment:
here it makes sense to call it defaultConnection and not 'default', since there is no context to know what 'default' it is.